extensions/gdx-setup/src/com/badlogic/gdx/setup/SetupPreferences.java

```diff
@@ -29,9 +29,30 @@
 import java.util.Map.Entry;
 import java.util.Properties;
 
-public class SetupPreferences {	
+public class SetupPreferences {
+
+	private static final File[] files = new File[] {
+		// $HOME/.gdxsetup
+		new File(new File(System.getProperty("user.home")), ".gdxsetup"),
+		// $CONFIG_HOME/gdxsetup/config
+		new File(new File(new File(findConfigHomePath()), "gdxsetup"), "config")};
```

Comment:
These should probably be in reverse order; because findFile() returns early if it finds a directory, and any users who have ever run gdx-setup in earlier versions will have a `.gdxsetup` folder in their home directory, the option to look in a configurable path is ignored for them.